openshift-operatorhub-kubecost-install.md

```diff
@@ -63,77 +63,202 @@ kubectl apply -f example-crd.yaml -n kubecost
 apiVersion: charts.kubecost.com/v1alpha1
 kind: CostAnalyzer
 metadata:
-  name: kubecost
+  name: cost-analyzer-ocp-sample
 spec:
-  # Default values copied from <project_dir>/helm-charts/cost-analyzer/values.yaml
-  openshiftDeployment: true
   affinity: {}
-  awsstore:
-    createServiceAccount: false
-    useAwsStore: false
-  clusterController:
-    enabled: false
-    image: gcr.io/kubecost1/cluster-controller:v0.1.0
-    imagePullPolicy: Always
-  extraVolumeMounts: []
-  extraVolumes: []
+  # Security Context settings for Redhat OpenShift cluster:
   kubecostProductConfigs:
-    clusterName: your-ocp-cluser
+    clusterName: YOUR_CLUSTER_NAME
+    # cloudIntegrationSecret: cloud-integration
+  kubecostDeployment:
+    podSecurityContext:
+    # Note: Un-comment these securityContext configs for OCP cluster 4.11+
+      # seccompProfile:
+      #   type: RuntimeDefault
+      runAsNonRoot: true
+  kubecostModel:
+    etlCloudAsset: true # set to true to enable kubecost to include out-of-cluster cloud resources  (uses more memory)
+    containerStatsEnabled: true
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true
+  # Note: Un-comment these securityContext configs for OCP cluster 4.11+
+      # seccompProfile:
+      #   type: RuntimeDefault
+      capabilities:
+        drop:
+          - ALL
+  kubecostFrontend:
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true
+  # Note: Un-comment these securityContext configs for OCP cluster 4.11+
+      # seccompProfile:
+      #   type: RuntimeDefault
+      capabilities:
+        drop:
+          - ALL
+  kubecostNetworkCosts:
+    securityContext: {}
+    containerSecurityContext: {}
+
   prometheus:
     nodeExporter:
       enabled: false
-    serviceAccounts:
-      nodeExporter:
-        create: false
```

Comment:
@linhlam-kc Won't removing the service account block cause the node exporter service account to be created?

Reply:
@srpomeroy Disabling nodeExporter should be enough according to this [condition](https://github.com/kubecost/openshift-helm-chart/blob/bc0e157f8d9376f93aaebe60ae300aaa0305c21a/cost-analyzer/charts/prometheus/templates/node-exporter-serviceaccount.yaml#L2) I do not see it is created in my testing last week as well on the OCP cluster. Let me try it real quick again.

Reply:
I can confirm that disable promethus.nodeExporter.enabled will also disable the creation of nodeExporter serviceAccount

Reply:
Awesome! There must have been a change. That was not the case previously.